lib/__tests__/geocodioLibraryNode.test.js

```diff
@@ -200,4 +200,35 @@ describe("geocoder", () => {
       expect(response.results.length).toEqual(1);
     });
   });
+
+  // LIST TESTS
+
+  it("Can create a new list, check status, download list and delete list", () => {
+    expect.assertions(1);
+
+    let newList = geocoder.list
+      .create(
+        `${__dirname}/stubs/sample_list.csv`,
+        "forward",
+        "{{A}} {{B}} {{C}} {{D}}",
+        "https://example.com/my-callback"
+      )
+      .then(response => {
+        expect(response.file.filename).toEqual("sample_list.csv");
+
+        geocoder.list.status(response.id).then(resp => {
+          expect(resp.id).toEqual(response.id);
+
```

Comment:
We need to either add a small delay here or keep checking the status until the file is completely processed

Reply:
Just to clarify, should I be delaying the running of the `.status()` function or the `expect()` method contained within? 

I just want to verify, because I know I would need to wrap the rest of the nested callback within `setTimeout()` if I use that to delay `status()`.